guides/source/routing.md

```diff
@@ -1108,32 +1182,27 @@ will recognize incoming paths beginning with `/photos` and route the requests to
 | PATCH/PUT | /photos/:id      | photos#update     | image_path(:id)      |
 | DELETE    | /photos/:id      | photos#destroy    | image_path(:id)      |
 
-### Overriding the `new` and `edit` Segments
+### Renaming the `new` and `edit` Helper Prefix
 
-The `:path_names` option lets you override the automatically-generated `new` and `edit` segments in paths:
+The `:path_names` option lets you override the default `new` and `edit` prefix in path helpers. For example:
```

Comment:
The title this sentence still claim that this changes helper prefixes. We should change that to "segment in paths" again